.storybook/stories/checkbox-toggle/checkbox-toggle.stories.ts

```diff
@@ -56,22 +41,83 @@ const defaultParameters: Parameters = {
   },
 };
 
-setupStorybook(ClrCheckboxModule, defaultStory, defaultParameters, generateVariants());
+const CheckBoxToggleTemplate: Story = args => {
+  const containerSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-container' : 'clr-toggle-container';
+  const wrapperSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-wrapper' : 'clr-toggle-wrapper';
+  const directive = args.type === CheckboxType.Checkbox ? 'clrCheckbox' : 'clrToggle';
```

Comment:
It looks like code was already repeated across files, but it's repeated even more now. Maybe put the helper function in my above comment in a `checkbox-toggle.helpers.ts` file.

Reply:
Moved this piece of code to a common helper function in `checkbox-toggle.helpers.ts`